wmt_shields/styles/wiki_symbol.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# This file is part of the Waymarked Trails Map Project
+# Copyright (C) 2023 Stéphane Chatty
+# 
+
+import sys
+from urllib.parse import urlparse, quote as urlquote
+
+import hashlib
+import re
+
+import gi
+gi.require_version('Rsvg', '2.0')
+from gi.repository import Rsvg
+import os
+
+from ..common.tags import Tags
+from ..common.config import ShieldConfig
+from ..common.shield_maker import ShieldMaker
+
+class WikiSymbol(ShieldMaker):
+    """ A shield with an SVG image located in the OSM wiki.
+    """
+
+    def __init__(self, uuid, url, config):
+        self.uuid_pattern = uuid
+        self.url = url
+        self.config = config
+
+    def render(self, ctx, w, h):
+        data = self.find_resource('', self.url)
```

Comment:
You need to protect against download failures here or imports will abort at most inconvenient times just because wiki.osm.org is temporarily down.